{"ops":[{"insert":"2022-03-22 09:41:51,831 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:51,870 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:52,936 31880 INFO 4G_INGENIERIA odoo.modules.registry: At least one model cache has been invalidated, signaling through the database."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:52,940 31880 INFO 4G_INGENIERIA werkzeug: 192.168.1.254 - - [22/Mar/2022 09:41:52] \"GET / HTTP/1.1\" 200 -"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,082 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,084 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,101 31880 INFO 4G_INGENIERIA odoo.addons.base.ir.ir_attachment: _read_file reading /opt/odoo/Odoo11/.local/share/Odoo/filestore/4G_INGENIERIA/af/af7cc08277ae9b6dcf9dce3618a181120d2083de"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/odoo/odoo/fields.py\", line 955, in __get__"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    value = record.env.cache.get(record, self)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/odoo/odoo/api.py\", line 977, in get"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    value = self._data[key][field][record._ids[0]]"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"KeyError: 49053"},{"attributes":{"code-block":true},"insert":"\n\n"},{"insert":"During handling of the above exception, another exception occurred:"},{"attributes":{"code-block":true},"insert":"\n\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/odoo/odoo/addons/base/ir/ir_attachment.py\", line 101, in _file_read"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    r = base64.b64encode(open(full_path,'rb').read())"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"FileNotFoundError: [Errno 2] No such file or directory: '/opt/odoo/Odoo11/.local/share/Odoo/filestore/4G_INGENIERIA/af/af7cc08277ae9b6dcf9dce3618a181120d2083de'"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,107 31880 INFO 4G_INGENIERIA werkzeug: 192.168.1.254 - - [22/Mar/2022 09:41:53] \"GET /web/image/49053/linkedin.png HTTP/1.1\" 200 -"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,178 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,189 31880 ERROR 4G_INGENIERIA odoo.addons.auth_session_timeout.models.res_users: Exception updating session file access/modified times."},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"Traceback (most recent call last):"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"  File \"/opt/odoo/Odoo11/extra_localization/auth_session_timeout/models/res_users.py\", line 100, in _auth_timeout_check"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"    utime(path, None)"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"PermissionError: [Errno 13] Permission denied"},{"attributes":{"code-block":true},"insert":"\n"},{"insert":"2022-03-22 09:41:53,269 31880 INFO 4G_INGENIERIA werkzeug: 192.168.1.254 - - [22/Mar/2022 09:41:53] \"POST /website/translations HTTP/1.1\" 200 -"},{"attributes":{"code-block":true},"insert":"\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n"},{"insert":"/opt/odoo/odoo"},{"attributes":{"code-block":true},"insert":"\n\n\n\n"},{"insert":"python3.7 odoo-bin --addons-path=/opt/odoo/Odoo11/addons,/opt/odoo/Odoo11/extra_localization/"},{"attributes":{"code-block":true},"insert":"\n\n"}]}